commands/help.rb

```diff
@@ -149,8 +149,6 @@ def self.help(command)
         Usage: crew upload [<package1> <package2> ...]
         This will update the binary_sha256 hashes in each package and upload binaries to GitLab.
         The GITLAB_TOKEN environment variable must be set to access the upstream repository.
-        If no package(s) are provided, all binaries in `#{CREW_LOCAL_REPO_ROOT}/release/<arch>` will be uploaded.
-        If `-v` or `--verbose` is present, additional debug information will be displayed.
```

Comment:
Why remove and require 3 separate uploads for all the architectures?

Reply:
The added code complexity wasn't worth it, and I couldn't think of a scenario where uploading all architectures was useful.

At any rate, if this is a functionality you do use, please see the PR description for how you can replicate it.